app/src/main/res/layout/loading_indicator.xml

```diff
@@ -7,6 +7,15 @@
     android:gravity="center"
     android:orientation="vertical">
 
+    <ImageView
+        android:src="@drawable/logotipas_nma_full"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:adjustViewBounds="true"
+        android:layout_marginEnd="50dp"
```

Comment:
`marginEnd` and `marginStart` are only supported since API level 17 and our app min API level is 16  https://github.com/vycius/NMAkademija/blob/master/app/build.gradle#L12 
Android studio should suggest fix for this.